pkg/apis/core/v1/defaults.go

```diff
@@ -349,6 +349,20 @@ func SetDefaults_HTTPGetAction(obj *v1.HTTPGetAction) {
 		obj.Scheme = v1.URISchemeHTTP
 	}
 }
+
+// SetDefaults_Namespace adds a default label for all namespaces
+func SetDefaults_Namespace(obj *v1.Namespace) {
```

Comment:
Can we add a unit test for this in `defaults_test.go`?

Reply:
ah ya, just saw, sure

Reply:
just added this unit testt. should be ok.  did an experiment in gitpod.io so will be interestingto see if this PR update worked :)